evm/src/recursive_verifier.rs

```diff
@@ -91,7 +98,8 @@ impl<T: Copy + Eq + PartialEq + Debug> PublicInputs<T> {
         };
         let challenger_state_before = next_chunk(&mut iter);
         let challenger_state_after = next_chunk(&mut iter);
-        let ctl_zs_last = iter.collect();
+        let ctl_zs_last: Vec<_> = iter.collect();
+        log::info!("from_vec num Zs: {}", ctl_zs_last.len()); // TODO
```

Comment:
What's the TODO for here?

Reply:
Oops I didn't mean to commit it, will remove.